ruby/read_file.rb

```diff
@@ -0,0 +1,23 @@
+"""Read a file from file path (given as a command line arg),
```

Comment:
Triple-quoted strings only work in Ruby as an illusion. Ruby syntax has a rarely-used feature that will concatenate consecutive string literals (even if they are using different string syntax):

```ruby
pp("one and" "two and"  "three")   # => "one andtwo andthree"
pp(%q[one and] "two and" 'three') # => "one andtwo andthree"
```

So, actually, in Ruby, a triple quoted string is actually a regular string with an empty string literal on either side
```ruby
pp("""triple quoted string""") # => "triple quoted string"
pp(""   'triple quoted string'   "") # => "triple quoted string"
```

As a consequence, a single quote inside what looks like a triple-quoted string will cause an error:
```ruby
""" hello "world"  """ # => Syntax error
```

I believe this feature exists to make multi-line strings easier:
```ruby
my_string = \
  "line 1\n" \
  "line 2\n" \
  "line 3\n"
```

Reply:
Cool good to know! Will update triple quotes to # comments